v1/format/structured_cloud_event.h

```diff
@@ -0,0 +1,24 @@
+#ifndef STRUCTURED_CLOUD_EVENT_H
+#define STRUCTURED_CLOUD_EVENT_H
+
+#include "proto/cloud_event.pb.h"
+
+namespace cloud_events {
+namespace format {
+
+enum CloudEventFormat {JSON};
+
+class StructuredCloudEvent {
+    private:
+        CloudEventFormat format_;
+        std::string serialized_cloud_event_;
+    public:
+        StructuredCloudEvent(CloudEventFormat format, std::string serialized_cloud_event);
+        CloudEventFormat GetCloudEventFormat();
+        std::string GetSerializedCloudEvent();
```

Comment:
If you decide to make this a class, you can name setters/getters just like the variable names (e.g. serialized_cloud_event()).

In these cases it is ok to have an inline definition in a header file.

https://google.github.io/styleguide/cppguide.html#Function_Names